vm/devices/get/guest_emulation_transport/src/error.rs

```diff
@@ -78,6 +78,8 @@ pub enum IgvmAttestError {
         input_size: usize,
         expected_size: usize,
     },
+    #[error("IGVM agent returned an error")]
```

Comment:
can this error message make it a bit more clear that this is explicitly a _generic_ error, so that whoever sees this message is (only slightly) less annoyed by the lack of additional context?

Reply:
Additional work is in the works to provide more visibility.  At the moment it is a generic error.

Reply:
The intent to make this error better in the future doesn't negate the need to clarify this current wording in the present